### ISSUE TITLE
example failed--where or how to generates the file train.list

### ISSUE THREAD

**Issue by SeekPoint** [rajarshd/ChainsofReasoning#7]
mldl@mldlUB1604:~/ub16_prj/ChainsofReasoning/run_scripts$ bash train.sh ./config.sh
experiment_dir ../
experiment_file ..//0.txt
output_dir results/lse
data_dir ../examples/data_small_output/_architecture_structure_address/
gpu_id -1
numEpoch 20
numEntityTypes 7
includeEntityTypes 1
includeEntity 0
numFeatureTemplates 10
 relationEmbeddingDim 250
entityTypeEmbeddingDim  100
entityEmbeddingDim  50
rnnHidSize 250
topK 2
K 5
Learning Rate 1e-3
Learning Rate Decay 0.0167
rnnType  rnn
epsilon 1e-8
gradClipNorm 5
gradientStepCounter 100000
saveFrequency 1
batchSize 32
useGradClip 1
package_path
useAdam 1
paramInit 0.1
evaluationFrequency 5
createExptDir 1
useReLU 1
l2 1e-3
rnnInitialization 1
regularize 0
numLayers 1
useDropout 0
relationVocabSize 51390
entityVocabSize 1542690
entityTypeVocabSize 2218
dropout 0.3
Executing:
th ..//model/OneModel.lua -dataDir ../examples/data_small_output/_architecture_structure_address/ -tokenFeatures 0 -minibatch 32 -gpuid -1 -learningRate 1e-3 -l2 1e-3 -numEpochs 20 -useAdam 1 -saveFrequency 1 -evaluationFrequency 5 -model results/lse/2017-07-05-23-01-50/_architecture_structure_address/model -rnnType rnn -exptDir results/lse/2017-07-05-23-01-50/_architecture_structure_address -relationVocabSize 51390 -entityTypeVocabSize 2218 -relationEmbeddingDim 250 -entityTypeEmbeddingDim 100 -numFeatureTemplates 10 -numEntityTypes 7 -includeEntityTypes 1 -includeEntity 0 -entityVocabSize 1542690 -entityEmbeddingDim 50 -rnnHidSize 250 -topK 2 -epsilon 1e-8 -gradClipNorm 5 -gradientStepCounter 100000 -useGradClip 1 -paramInit 0.1 -createExptDir 1 -useReLU 1 -rnnInitialization 1 -learningRateDecay 0.0167 -regularize 0 -numLayers 1 -useDropout 0 -dropout 0.3 -K 5
Log file is results/lse/2017-07-05-23-01-50/_architecture_structure_address/log.txt
nn.Sequential {
  [input -> (1) -> (2) -> (3) -> (4) -> (5) -> (6) -> output]
  (1): nn.SplitTable
  (2): nn.Sequential {
    [input -> (1) -> (2) -> output]
    (1): nn.ConcatTableNoGrad {
      input
        |`-> (1): nn.Sequential {
        |      [input -> (1) -> (2) -> (3) -> output]
        |      (1): nn.NarrowTable
        |      (2): nn.ParallelTable {
        |        input
        |          |`-> (1): nn.LookupTable
        |          |`-> (2): nn.LookupTable
        |          |`-> (3): nn.LookupTable
        |          |`-> (4): nn.LookupTable
        |          |`-> (5): nn.LookupTable
        |          |`-> (6): nn.LookupTable
        |           `-> (7): nn.LookupTable
        |           ... -> output
        |      }
        |      (3): nn.CAddTable
        |    }
        |`-> (2): nn.Sequential {
        |      [input -> (1) -> (2) -> output]
        |      (1): nn.SelectTable(-1)
        |      (2): nn.LookupTable
        |    }
         ... -> output
    }
    (2): nn.JoinTable
  }
  (3): nn.SplitTable
  (4): nn.Sequencer @ nn.Recurrence @ nn.MaskZero @ nn.Sequential {
    [input -> (1) -> (2) -> (3) -> output]
    (1): nn.ParallelTable {
      input
        |`-> (1): nn.Linear(350 -> 250)
         `-> (2): nn.Linear(250 -> 250)
         ... -> output
    }
    (2): nn.CAddTable
    (3): nn.ReLU
  }
  (5): nn.SelectTable(-1)
  (6): nn.Linear(250 -> 46)
}
Reducer is LogSumExp	
reading file list from ../examples/data_small_output/_architecture_structure_address/train.list	
/home/mldl/torch/install/bin/luajit: ../model/batcher/BatcherFileList.lua:20: bad argument #1 to 'lines' (../examples/data_small_output/_architecture_structure_address/train.list: No such file or directory)
stack traceback:
	[C]: in function 'lines'
	../model/batcher/BatcherFileList.lua:20: in function '__init'
	/home/mldl/torch/install/share/lua/5.1/torch/init.lua:91: in function </home/mldl/torch/install/share/lua/5.1/torch/init.lua:87>
	[C]: in function 'BatcherFileList'
	..//model/OneModel.lua:318: in main chunk
	[C]: in function 'dofile'
	...mldl/torch/install/lib/luarocks/rocks/trepl/scm-1/bin/th:150: in main chunk
	[C]: at 0x00405d50
mldl@mldlUB1604:~/ub16_prj/ChainsofReasoning/run_scripts$

**Comment by SeekPoint** [comment]
????

**Comment by rajarshd** [comment]
I am not sure what you mean, but all the lines are struck through. I have also decided that I wont maintain the lua code base anymore. A pytorch version would be coming out hopefully soon
